### DISCUSSION TITLE
Menu button padre en un colapsable pueda ser Link

### DISCUSSION THREAD

**Discussion by hrchioest** [TiendaNube/nimbus-patterns#50] [Q&A] [answered]
Buenas tardes, estuve trabajando cambiando el menu para que los botones sean links y el único detalle que faltaría completar es que en os botones colapsables, el botón padre también puedan ser link, asi podamos tener el mismo comportamiento que en el old admin. Ya se lo comunique a @juniorconquista pero hago extensivo la solicitud y ayuda para cerrar este[ issue](https://tiendanube.atlassian.net/browse/NA-2649) 
Desde ya gracias!

**Comment by juniorconquista** [comment, marked as answer]
@hrchioest estou encerrando essa discussion pois o tema ja foi resolvido, adicionamos essa opção ao MenuButton.
